maisim/maisim.Game/Graphics/UserInterface/Overlays/SettingsSection.cs

```diff
@@ -0,0 +1,104 @@
+using maisim.Game.Graphics.Sprites;
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Localisation;
+using osuTK;
+using osuTK.Graphics;
+
+namespace maisim.Game.Graphics.UserInterface.Overlays
+{
+    public abstract class SettingsSection : Container
+    {
+        protected FillFlowContainer FlowContent;
+        protected override Container<Drawable> Content => FlowContent;
+
+        private IBindable<SettingsSection> selectedSection;
```

Comment:
Unused (as per CI report)

Reply:
Yes just try something but forget to remove it